controllers/uploadpackage.lua

```diff
@@ -0,0 +1,108 @@
+local app_helpers = require("lapis.application")
+local validate = require("lapis.validate")
+local capture_errors, assert_error = app_helpers.capture_errors, app_helpers.assert_error
+local lfs = require('lfs')
+local Packages = require("models.packages")
+local Users = require("models.users")
+
+local function isFile(name)
+  if not isDir(name) then
+    return os.rename(name,name) and true or false
```

Comment:
I think this warrants a comment explaining why is it doing this

Reply:
Added a brief comment explaining why we are using this particular hacky bit of hackiness.